### ISSUE TITLE
Discuss and reach an architecture for MI-CI 

### ISSUE THREAD

**Issue by matjohn2** [CiscoCloud/microservices-infrastructure#547]
We need do decide how to break down issue #163 into more achievable stories/tasks before we start work on building the MI-CI system

**Comment by siddharthist** [comment]
We have a plan, it is to use Drone to build on one cloud at a time. Closing in favor of #874.
